llvm-external-projects/iree-dialects/lib/Dialect/Input/InputOps.cpp

```diff
@@ -154,14 +154,12 @@ Value TiedOpInterface::findTiedBaseValue(Value derivedValue) {
 }
 
 bool TiedOpInterface::hasAnyTiedUses(Value value) {
-  for (auto &use : value.getUses()) {
-    auto tiedOp = dyn_cast<TiedOpInterface>(use.getOwner());
-    if (!tiedOp)
-      continue;
-    if (tiedOp.isOperandTied(use.getOperandNumber()))
-      return true;
-  }
-  return false;
+  return llvm::any_of(value.getUses(), [](auto &use) {
+    if (auto tiedOp = dyn_cast<TiedOpInterface>(use.getOwner())) {
```

Comment:
dyn_cast_if_present ? (I think if this is block arg then we'd be operating on nullptr with dyn_cast)

Reply:
Uses are always ops so it should be safe